web/src/components/pages/partner/partner-options.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+export const PARTNER_OPTIONS = [
+  {
+    id: 'community',
+    elems: {
+      about: <a href="about">About</a>,
+      community: (
+        <a
+          href="https://common-voice.github.io/community-playbook"
+          target="_blank"
+          rel="noreferrer">
+          Community playbook
+        </a>
+      ),
+      emailFragment: (
+        <a
+          href="mailto:commonvoice@mozilla.com?subject=Community partnership"
+          target="_blank"
+          rel="noreferrer">
+          get in touch.
+        </a>
+      ),
+    },
+  },
+  {
+    id: 'foundations',
+    elems: {
+      programmaticWork: (
+        <a
+          href="https://foundation.mozilla.org/en/common-voice/in-country-programmes"
+          target="_blank"
+          rel="noreferrer">
+          programmatic work
```

Comment:
Will this be translated?

Reply:
Yeah, I think it will. Thanks for pointing it out